src/commands/database-rules-canary.ts

```diff
@@ -0,0 +1,23 @@
+import { Command } from "../command";
+import * as logger from "../logger";
+import * as requireInstance from "../requireInstance";
+import { requirePermissions } from "../requirePermissions";
+import * as metadata from "../database/metadata";
+
+export default new Command("database:rules:canary <ruleset_id>")
+  .description("mark a staged ruleset as the canary ruleset")
```

Comment:
Where is it enforced that the canary ruleset has to be the staged ruleset?

Reply:
A staged ruleset is any ruleset that has an id, so it is enforce structurally.